src/planscape/plan/models.py

```diff
@@ -23,3 +24,43 @@ class Plan(models.Model):
 
     # The planning area of the plan.
     geometry = models.MultiPolygonField(srid=4269, null=True)
+
+class Project(models.Model): 
+    """
+    A Project is associated with one User, the owner, and one Plan. It has optional user-specified
+    project parameters, e.g. constraints.
+    """
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, null=True) 
```

Comment:
i think we decided that a plan should always have an owner so this should be false

Reply:
Maybe add a TODO instead? we want alpha users to not be signed in, hence the "null=True" condition.

Reply:
Done